Sample/O365-iOS-Snippets/Office365Snippets.m

```diff
@@ -425,6 +423,92 @@ - (void)deleteCalendarEvent:(MSOutlookEvent *)event
 
 }
 
+//Accepts an event with comment - comment can be nil
+- (void)acceptCalendarEvent:(MSOutlookEvent *)event
+                withComment:(NSString*)comment
+                 completion:(void (^)(BOOL success, NSError *error))completion{
+    Office365ClientFetcher *clientFetcher = [[Office365ClientFetcher alloc]init];
+    
+    [clientFetcher fetchOutlookClient:^(MSOutlookClient *client) {
+        MSOutlookUserFetcher *userFetcher = [client getMe];
+        MSOutlookEventFetcher *eventFetcher = [userFetcher getEventsById:event.Id];
+        MSOutlookEventOperations *operations = [eventFetcher operations];
+
+        NSURLSessionTask *task = [operations acceptWithComment:comment callback:^(int returnValue, MSODataException *error) {
+                BOOL success = (returnValue == 0);
+                completion(success, error);
+        }];
+        [task resume];
+    }];
+}
+
+//Declines an event with comment - comment can be nil
+- (void)declineCalendarEvent:(MSOutlookEvent *)event
+                 withComment:(NSString*)comment
+                  completion:(void (^)(BOOL success, NSError *error))completion{
+    Office365ClientFetcher *clientFetcher = [[Office365ClientFetcher alloc]init];
+    
+    [clientFetcher fetchOutlookClient:^(MSOutlookClient *client) {
+        MSOutlookUserFetcher *userFetcher = [client getMe];
+        MSOutlookEventFetcher *eventFetcher = [userFetcher getEventsById:event.Id];
+        MSOutlookEventOperations *operations = [eventFetcher operations];
+        
+        NSURLSessionTask *task = [operations declineWithComment:comment callback:^(int returnValue, MSODataException *error) {
+            BOOL success = (returnValue == 0);
+            completion(success, error);
+        }];
+        [task resume];
+    }];
+}
+
+//Tentatively accepts an event with comment - comment can be nil
+- (void)tentativelyAccept:(MSOutlookEvent *)event
+              withComment:(NSString*)comment
+               completion:(void (^)(BOOL success, NSError *error))completion{
+    Office365ClientFetcher *clientFetcher = [[Office365ClientFetcher alloc]init];
+    
+    [clientFetcher fetchOutlookClient:^(MSOutlookClient *client) {
+        MSOutlookUserFetcher *userFetcher = [client getMe];
+        MSOutlookEventFetcher *eventFetcher = [userFetcher getEventsById:event.Id];
+        MSOutlookEventOperations *operations = [eventFetcher operations];
+        
+        NSURLSessionTask *task = [operations tentativelyAcceptWithComment:comment callback:^(int returnValue, MSODataException *error) {
+            BOOL success = (returnValue == 0);
+            completion(success, error);
+        }];
+        [task resume];
+    }];
+}
+
+// Fetches calendar view in specified date range
+// For more information about calendar view, visit https://msdn.microsoft.com/office/office365/APi/calendar-rest-operations#GetCalendarView
```

Comment:
General comment: Nice! We need to link more often documentation. We should consider this throughout our snippets.